example_cpp_smart_card_client_app/src/background.js

```diff
@@ -213,4 +213,8 @@ if (GSC.Packaging.MODE === GSC.Packaging.Mode.APP) {
 
 // Automatically load the App (in the background) with Chrome startup.
 GSC.AppUtils.enableSelfAutoLoading();
+
+// Trigger the PC/SC test.
+executableModule.getMessageChannel().send(
```

Comment:
This change is orthogonal to the PR - it's not about the SHA256. And actually, it's not important to be merged into `main` either. (It was just a quick hack to check whether PC/SC-Lite calls are still possible from the examples built the `PACKAGING=extension` mode.)